lib/panoptes/client.rb

```diff
@@ -60,7 +60,7 @@ def initialize(env: :production, auth: {}, public_key_path: nil, params: {})
     def current_user
       raise NotLoggedIn unless @auth[:token]
 
-      payload, = JWT.decode @auth[:token], jwt_signing_public_key, algorithm: 'RS512'
+      payload = JWT.decode @auth[:token], jwt_signing_public_key, algorithm: 'RS512'
```

Comment:
That was totally not clear enough on my part. `JWT.decode` returns an array, and that comma was there to unpack it. `payload, _ = JWT....` would have been much clearer.

Reply:
With or without that comma it fails the specs, but I'll add it back in now.